### PR TITLE
Added Policy Based Route actions.

### PR DESCRIPTION
**Pull request by yujunliang**

### CODE REVIEW

File: chaosgcp/networkconnectivity/actions.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    # Create a client
+    client = networkconnectivity_v1.PolicyBasedRoutingServiceClient()
+
+    # Initialize request argument(s)
+    request = networkconnectivity_v1.DeletePolicyBasedRouteRequest(
+        name=name_value,
+    )
+
+    # Make the request
+    client.delete_policy_based_route(request=request)
```
Comment (Lawouach) [inline chaosgcp/networkconnectivity/actions.py]: Make sure to return the response as per the signature of the function

```python
from chaosgcp import to_dict
```

```python
response = client.delete_policy_based_route(request=request)

return to_dict(response)
```

**Review by Lawouach — CHANGES_REQUESTED** [review]
This looks good. 

However you need to add a CHANGELOG entry please.

File: chaosgcp/networkconnectivity/actions.py (hunk 1 above)
Comment (yujunliang) [reply chaosgcp/networkconnectivity/actions.py]: Added.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,10 +6,11 @@
 
 ## [0.24.0][] - 2024-04-30
 
-[0.23.0]: https://github.com/chaostoolkit-incubator/chaostoolkit-google-cloud-platform/compare/0.23.0...0.24.0
+[0.24.0]: https://github.com/chaostoolkit-incubator/chaostoolkit-google-cloud-platform/compare/0.23.0...0.24.0
```
Comment (Lawouach) [inline CHANGELOG.md:9]: good catch

**Review by Lawouach — APPROVED** [review]
LGTM

**Pull request merged**